mwdb/resources/remotes.py

```diff
@@ -0,0 +1,459 @@
+from flask_restful import Resource
+from flask import g
+import requests
+import json
+from mwdb.core.plugins import hooks
+from werkzeug.exceptions import BadRequest, Conflict, NotFound, Forbidden, InternalServerError
+from mwdb.core.capabilities import Capabilities
+from mwdb.core.config import app_config
+from mwdb.schema.remotes import APIRemotesListResponseSchema
+from mwdb.model import db, File, Config, TextBlob, Group
+from mwdb.model.object import ObjectTypeConflictError
+from mwdb.schema.file import FileItemResponseSchema
+from mwdb.schema.config import ConfigItemResponseSchema
+from mwdb.schema.blob import BlobItemResponseSchema
+from tempfile import SpooledTemporaryFile
+from . import access_object, logger, requires_authorization
+
+
+class APIRemoteListResource(Resource):
+    @requires_authorization
+    def get(self):
+        """
+        ---
+        summary: Get list of configured remote names
+        description: |
+            Return a list of available remote names
+        security:
+            - bearerAuth: []
+        tags:
+            - api_remote
+        responses:
+            200:
+                description: List of user configured remotes
+                content:
+                  application/json:
+                    schema: GroupListResponseSchema
```

Comment:
Wrong schema in documentation
```suggestion
                    schema: APIRemotesListResponseSchema
```